management/terraform_min_version.rego

```diff
@@ -0,0 +1,14 @@
+package terraform
+
+import input.tfplan as tfplan
+
+min_version = "1.0.0"
+
+# Matches that minimum version is met
+deny[reason] {
+	version := tfplan.terraform_version
+
+	version < min_version
```

Comment:
This better be done using built-in function for semantic versions comparison:
```suggestion
	semver.compare(version, min_version) < 0
```

Reply:
Thanks, I wasn't aware of that function.